Rubberduck.Parsing/Inspections/Resources/InspectionsUI.resx

```diff
@@ -839,4 +839,13 @@ If the parameter can be null, ignore this inspection result; passing a null valu
   <data name="RemoveEmptyConditionBlockQuickFix" xml:space="preserve">
     <value>Remove empty If and Else blocks</value>
   </data>
+  <data name="ShadowedDeclarationInspectionMeta" xml:space="preserve">
+    <value>Two declarations are in scope and have the same identifier name. This means that only one of them will be available to use.</value>
+  </data>
+  <data name="ShadowedDeclarationInspectionName" xml:space="preserve">
+    <value>Shadowed declaration</value>
+  </data>
+  <data name="ShadowedDeclarationInspectionResultFormat" xml:space="preserve">
+    <value>{0} '{1}' hides {2} '{3}'</value>
+  </data>
```

Comment:
how did these get in here?

Reply:
Likely from a `git pull` to sync with RD.